client/src/components/dashboard/Chat/ChatController.js

```diff
@@ -168,17 +168,20 @@ class ChatController extends React.Component {
         <Modal size="large" open={this.state.modal} close={this.toggleModal} />
         <div>
           <div className="ui comments">
-            <h2 className="ui dividing header" style={ !screen.isDesktop ? { width: '200px' } : null }>
+            <h2 className="ui dividing header" style={ !screen.isDesktop ? { width: 200 } : null }>
           { conversant
-            ? <span style={ !screen.isDesktop ? { fontSize: '16px' } : null }>
+            ? <span style={ !screen.isDesktop ? { fontSize: 16 } : null }>
                 <img src={this.props.conversantAvatar} className='privateAvatar' alt={`${conversant}'s Avatar'`} />
                 Private Chat with&nbsp;
                 <span
                   className='conversant'
                   onClick={() => this.props.history.replace(`/dashboard/profile/${conversant}`)}>
-                   {conversant} </span>
+                  {conversant}
+                </span>
+              { !onlineStatus.has(conversant) &&
+                <span style={{ fontSize: 16, marginLeft: 5 }}>(offline)</span> }
               </span>
```

Comment:
@bonham000 really hard to see the changes because of this red, but they are right here ☝️ 

Here's a screenshot of the code below. The other changes are just changing style objects from `{ fontSize: '16px' }` to `{ fontSize: 16 }` since React supports that and to keep things consistent throughout the app. 

Lines 181 + 182:
![image](https://cloud.githubusercontent.com/assets/18563015/25603455/f0e4369a-2ec9-11e7-8192-db8661bb612a.png)